.babelrc

```diff
@@ -0,0 +1,21 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": false
+      }
+    ],
+    "@babel/preset-typescript",
+    "@babel/preset-react",
+    "@emotion/babel-preset-css-prop"
+  ],
+  "plugins": [
+    [
+      "@babel/plugin-transform-runtime",
+      {
+        "corejs": 3
```

Comment:
why need this? (Iremember I've seen something about that in the different issues)

Reply:
This should polyfill without polluting the globals. Probably better for a library like this one?

Reply:
From my understanding:
- babel allow to use core-js 2 or 3
- babel uses core-js 2 as a default (dunno why)
- core-js 2 doesn't support Promise.any (https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md)

Looks like only this line was needed to fix the issue? Can confirm?

Reply:
Yea, you could say so. For v2, I think we need to install babel polyfill for that but the repo says it's deprecated already. The v3 comes with promise.any which is more convenient.